src/patches/key.h

```diff
@@ -0,0 +1,32 @@
+#pragma once
+
+#include <absl/strings/ascii.h>
+#include <map>
+#include <prime/KeyCode.h>
+#include <string>
+#include <vector>
+
+class Key
+{
+private:
+  static const std::map<std::string, KeyCode> mappedKeys;
+  static const std::vector<KeyCode>           modifiers;
```

Comment:
We could probably get away with mapping this to a bit mask, that way we wouldn't have to do a linear scan to find the modifier.
Of course, that will require the other work I've mentioned to be in-place with regards to caching key-states per-frame somewhere.

Reply:
As an additional note. This is entirely static class. I would prefer to use a namespace instead for that.

Reply:
Not sure what you mean on the namespace front, but I have now removed the modifiers in favour of static code.